e2e/kubectl.e2e.spec.ts

```diff
@@ -0,0 +1,117 @@
+import path from 'path';
+import {
+  ElectronApplication, BrowserContext, _electron, Page, Locator
+} from 'playwright';
+import { test, expect } from '@playwright/test';
+import { TestUtils } from './utils/TestUtils';
+import * as tools from './utils/ToolsUtils';
+
+let page: Page;
+const defaultReportFolder = path.join(__dirname, 'reports/');
+
+test.describe.serial('Rancher Desktop - K8s Deploy Test', () => {
+  let mainTitle: Locator;
+  let utils: TestUtils;
+  let electronApp: ElectronApplication;
+  let context: BrowserContext;
+
+  const mainTitleSelector = '[data-test="mainTitle"]';
+
+  test.beforeAll(async() => {
+    utils = new TestUtils();
+    utils.createDefaultSettings();
+
+    electronApp = await _electron.launch({
+      args: [
+        path.join(__dirname, '../'),
+        '--disable-gpu',
+        '--whitelisted-ips=',
+        '--disable-dev-shm-usage',
+      ]
+    });
+    context = electronApp.context();
+
+    await context.tracing.start({ screenshots: true, snapshots: true });
+    page = await electronApp.firstWindow();
+  });
+
+  test.afterAll(async() => {
+    await context.tracing.stop({ path: `${ defaultReportFolder }pw-trace.zip` });
```

Comment:
I've created a variable using `path.join()` at the top of the spec file.
`const defaultReportFolder = path.join(__dirname, 'reports/');`

If we need to change the default report folder, we just need to modify a single line. Is it ok or there's a better approach for it? Any suggestions will be very appreciated.

Reply:
No, the problem here is that you're joining `…/reports/` as text to `pw-trace.zip`.  We should use `path.join(defaultReportFolder, 'pw-trace.zip');` instead.  (Because, on Windows, the proper output is `C:\…\reports\pw-trace.zip`.)

How `defaultReportFolder` itself is constructed is okay.

Reply:
Shooot, you're right... totally my bad.
I'll fix that up :)